packages/wonder-blocks-core/src/components/render-state-context.ts

```diff
@@ -22,7 +22,9 @@ export const RenderState = {
  * standard:
  *   means that we're all now doing non-SSR rendering
  */
-// @ts-expect-error [FEI-5019] - TS2322 - Type 'Context<"root">' is not assignable to type 'Context<"initial" | "root" | "standard">'.
```

Comment:
**note:** I fixed this typing issue which also removed suppressions in a test file.